web/src/lib/auth/signinHelper.ts

```diff
@@ -11,7 +11,7 @@ import {
   setRegistrationDimension,
   setUserId,
 } from "@/lib/utils/analytics-helpers";
-import { createEmptyUser } from "@businessnjgovnavigator/shared/";
+import { createEmptyUser, UserData } from "@businessnjgovnavigator/shared/";
 import { Dispatch } from "react";
 import { UpdateQueue } from "../types/types";
 import { AuthAction } from "./AuthContext";
```

Comment:
I know you didn't do this, but can we swap relative imports?
```suggestion
import { UpdateQueue } from "@/lib/types/types";
import { AuthAction } from "@/lib/auth/AuthContext";
import * as session from "@/lib/auth/sessionHelper";
import { triggerSignOut } from "@/lib/auth/sessionHelper";
```